test/testutils.js

```diff
@@ -428,32 +428,34 @@ testUtils.isIE11 = (function isIE11(navigator) {
 
 axe.testUtils = testUtils;
 
-beforeEach(function() {
-  // reset from axe._load overriding
-  checks = originalChecks;
-  axe._audit = originalAudit;
-  axe._audit.rules = originalRules;
-  commons = axe.commons = originalCommons;
-});
-
-afterEach(function() {
-  axe.teardown();
-  fixture.innerHTML = '';
-
-  // remove all attributes from fixture (otherwise a leftover
-  // style attribute would cause avoid-inline-spacing integration
-  // test to fail with [#fixture] being included in the results)
-  var attrs = fixture.attributes;
-  for (var i = 0; i < attrs.length; i++) {
-    var attrName = attrs[i].name;
-    if (attrName !== 'id') {
-      fixture.removeAttribute(attrs[i].name);
+if (typeof beforeEach !== 'undefined' && typeof afterEach !== 'undefined') {
```

Comment:
I put this in so that we can use testutils outside a test run. I was using it in playground.